package.json

```diff
@@ -52,6 +52,7 @@
     "downshift": "^5.4.2",
     "ngreact": "^0.5.0",
     "query-string": "^6.1.0",
+    "react-autocomplete": "^1.8.1",
```

Comment:
I think this will need an ok from Packaging. Also, this component seems to be [no longer maintained](https://github.com/reactjs/react-autocomplete).  How painful of an effort would it be to use Patternfly components instead?

Reply:
Are you referring to `SearchInput`?

Reply:
I'm referring to the `Autocomplete` component which is imported here from `react-autocomplete`. Can this be done without bringing in any new libraries (e.g., with no changes to `package.json`)?

Reply:
As for what existing Patternfly component(s) will replace it, I haven't researched that; I'll leave that up to you to tell me if it's doable or not.

Reply:
Seems Patternfly doesn't have an equivalent component for `react-autocomplete`. 
`SearchInput` is the best without a new react library. 
I would suggect https://github.com/JedWatson/react-select if current implementation with `SearchInput` does not look good.